nservicebus/azure/azure-servicebus-transport.md

```diff
@@ -112,18 +113,37 @@ snippet: ASB-NamingConventions-entity-creation-conventions
 
 WARNING: This is an advanced topic and requires full understanding of the topology.
 
+## Transactions and delivery guarantees
 
-## Transactions
+NServiceBus AzureServiceBus transport relies on the underlying Azure ServiceBus library which requires the use of the `Serializable` isolation level (the most restrictive isolation level that does not permit `dirty reads`, `phantom reads` and `non repeatable reads`; will block any reader until the writer is committed. For more information refer to [Transaction Isolation Levels Explained in Details](http://dotnetspeak.com/2013/04/transaction-isolation-levels-explained-in-details) article.
 
-NServiceBus AzureServiceBus transport relies on the underlying Azure ServiceBus library which requires the use of the `Serializable` isolation level (the most restrictive isolation level that does not permit `dirty reads`, `phantom reads` and `non repeatable reads`; will block any reader until the writer is committed [see this link](http://dotnetspeak.com/2013/04/transaction-isolation-levels-explained-in-details) for more information)
+NServiceBus AzureServiceBus transport configuration is hard-coded to `Serializable` isolation level. Users can't override it.
 
-NServiceBus AzureServiceBus transport configuration is hard coded to `Serializable` isolation level to prevent users from overriding it.
+### Version 6 and above
+Azure Service Bus Transport supports `SendAtomicWithReceive`, `ReceiveOnly` and `Unreliable` levels.
 
+#### SendAtomicWithReceive
+Note: `SendAtomicWithReceive` level is supported only when destination and receive queues are in the same namespace.
 
-## Scenarios
+The `SendAtomicWithReceive` guarantee is achieved by using `ViaEntityPath` property on outbound messages. It's value is set to the receiving queue.
+
+If the `ViaEntityPath` is not empty, then messages will be added to the receive queue. The messages will be forwarded to their actual destination (inside the broker) only when the complete operation is called on the received brokered message. The message won't be forwarded if the 30 seconds operation limit is exceeded or if the message is explicitly abandoned.
+
+#### ReceiveOnly
 
-For any scenario provided in this document, you should always test it in environment as close to production as possible.
+The `ReceiveOnly` guarantee is based on the Azure Service Bus Peek-Lock mechanism. 
+
+The message is not removed from the queue directly after receive, but it's hidden for 30 seconds. That prevents other instances from picking it up. If the receiver fails to process the message withing that timeframe or explicitly abandons the message, then the message will become visible again. Other instances will be able to pick it up.
```

Comment:
> hidden for 30 seconds. 

Not accurate. 30 seconds is the default. It's "hidden" for _lock duration_ time.